packages/plugins/typescript/vue-apollo/src/visitor.ts

```diff
@@ -16,6 +16,7 @@ export interface VueApolloPluginConfig extends ClientSideBasePluginConfig {
   vueApolloComposableImportFrom: 'vue' | '@vue/apollo-composable' | string;
   vueCompositionApiImportFrom: 'vue' | '@vue/apollo-composable' | string;
   addDocBlocks: boolean;
+  clientId: string | null;
```

Comment:
we don't want a required prop given that can be a breaking changes for users.
```suggestion
  clientId?: string | null;
```